### PR TITLE
Replaced logic to bind to all addresses on 0.0.0.0

### PR DESCRIPTION
**Pull request by amroessam**

### CODE REVIEW

**Comment by amroessam** [comment]
https://github.com/bk138/droidVNC-NG/issues/14

File: .idea/compiler.xml [hunk 1]
```diff
@@ -11,6 +11,6 @@
       <entry name="!?*.kt" />
       <entry name="!?*.clj" />
     </wildcardResourcePatterns>
-    <bytecodeTargetLevel target="1.8" />
+    <bytecodeTargetLevel target="11" />
```
Comment (bk138) [inline .idea/compiler.xml:14]: Please don't change this file as it's not related.

File: .idea/gradle.xml [hunk 2]
```diff
@@ -4,11 +4,10 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
+        <option name="delegatedBuild" value="true" />
```
Comment (bk138) [inline .idea/gradle.xml:7]: Please don't change this file as it's not related.

File: .idea/misc.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
       <configuration PROFILE_NAME="Debug" CONFIG_NAME="Debug" />
     </configurations>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="11" project-jdk-type="JavaSDK">
```
Comment (bk138) [inline .idea/misc.xml:8]: Please don't change this file as it's not related.

File: app/src/main/java/net/christianbeier/droidvnc_ng/MainService.java [hunk 4]
```diff
@@ -414,28 +414,9 @@ public static String getIPv4AndPort() {
             SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(instance);
             port = prefs.getInt(Constants.PREFS_KEY_SETTINGS_PORT, 5900);
         } catch (NullPointerException e) {
-            //unused
```
Comment (bk138) [inline app/src/main/java/net/christianbeier/droidvnc_ng/MainService.java:417]: please don't change unrelated lines

File: app/src/main/java/net/christianbeier/droidvnc_ng/MainService.java [hunk 5]
```diff
@@ -414,28 +414,9 @@ public static String getIPv4AndPort() {
...
-                            return ia.getAddress().toString().replaceAll("/", "") + ":" + port;
-                        }
-                    }
-                }
-            }
-        } catch (SocketException e) {
-            //unused
-        }
-        return null;
+        return "0.0.0.0:" + port;
```
Comment (bk138) [inline app/src/main/java/net/christianbeier/droidvnc_ng/MainService.java:419]: In fact, this is the only change needed for this pull request.

**Review by bk138 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**